pandas/core/base.py

```diff
@@ -500,7 +500,13 @@ def array(self) -> ExtensionArray:
         """
         raise AbstractMethodError(self)
 
-    def to_numpy(self, dtype=None, copy=False, na_value=lib.no_default, **kwargs):
+    def to_numpy(
+        self,
+        dtype: Optional[Dtype] = None,
```

Comment:
NpDtype

Reply:
On L664 we do `if is_extension_array_dtype(self.dtype)`, therefore `Dtype` seemed more appropriate than `NpDtype`, cause the function does accept dtypes other the one from numpy.